sdk/test/UnitTests/Custom/Runtime/SignerTests.cs

```diff
@@ -24,15 +24,14 @@ public void TestSignerWithAnonymousCredentials()
         {            
             var pipeline = new RuntimePipeline(new MockHandler());
             pipeline.AddHandler(new Signer());
-            pipeline.AddHandler(new CredentialsRetriever());
 
             var signer = new MockSigner();
             var context = CreateTestContext(signer);
             context.RequestContext.Identity = new AnonymousAWSCredentials();
             pipeline.InvokeSync(context);
 
             Assert.IsTrue(context.RequestContext.IsSigned);
-            Assert.AreEqual(0, signer.SignCount);
+            Assert.AreEqual(1, signer.SignCount);
```

Comment:
Calling any an action with `AnonymousAWSCredentials` now goes through the signing phase and calls the `NullSigner`.